signer/src/utxo.rs

```diff
@@ -576,10 +644,10 @@ mod tests {
     const PUBLIC_KEY1: &'static str =
         "032e58afe51f9ed8ad3cc7897f634d881fdbe49a81564629ded8156bebd2ffd1af";
 
-    const XONLY_PUBLIC_KEY0: &'static str =
+    const X_ONLY_PUBLIC_KEY0: &'static str =
         "ff12471208c14bd580709cb2358d98975247d8765f92bc25eab3b2763ed605f8";
 
-    const XONLY_PUBLIC_KEY1: &'static str =
+    const X_ONLY_PUBLIC_KEY1: &'static str =
         "2e58afe51f9ed8ad3cc7897f634d881fdbe49a81564629ded8156bebd2ffd1af";
```

Comment:
I assume reusing the same key across multiple integration tests creates one source of interference preventing us from running them in parallel. Would it be easy to add a function that generates this key from an rng?.

Reply:
No this is completely unrelated. None of the tests in this file are integration tests.

Reply:
Oh, then nevermind :sweat_smile: